src/Symfony/Component/AssetMapper/Tests/ImportMap/ImportMapRendererTest.php

```diff
@@ -132,7 +132,7 @@ public function testCustomScriptAttributes()
         ]);
         $html = $renderer->render([]);
         $this->assertStringContainsString('<script type="importmap" something data-turbo-track="reload">', $html);
-        $this->assertStringContainsString('<script async src="https://polyfillUrl.example" something data-turbo-track="reload"></script>', $html);
```

Comment:
you lost the assertion on `data-turbo-track="reload"`

Reply:
oops, fixed